### PR TITLE
show list books

### PR DESCRIPTION
**Pull request by HuyLuuVan**
Ready 

+ Show list books complete
+ https://edu-redmine.sun-asterisk.vn/issues/26118

### CODE REVIEW

**Comment by BuBuHuy** [comment]
Ready 

+ Show list books complete
+ https://edu-redmine.sun-asterisk.vn/issues/26118

**Comment by vanvtt-0952** [comment]
commented
pull này 3 commit, rebase thành 1 đi

**Comment by HuyLuuVan** [comment]
+ Đã Rebase chị ơi

**Comment by HuyLuuVan** [comment]
Ready 

+ Show list books complete
+ https://edu-redmine.sun-asterisk.vn/issues/26118

p/s : 1 buổi sáng toàn conflict 😭😭

File: config/environments/production.rb [hunk 1]
```diff
@@ -53,14 +73,22 @@
   config.log_level = :debug
 
   # Prepend all log lines with the following tags.
+<<<<<<< HEAD
   config.log_tags = [:request_id]
+=======
```
Comment (vanvtt-0952) [inline config/environments/production.rb:78]: ...

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -4,15 +4,15 @@
     <title>Library</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
-    <%= javascript_pack_tag "application", "data-turbolinks-track": "reload" %>
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (vanvtt-0952) [inline app/views/layouts/application.html.erb:7]: nháy đôi

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                                            <% else %>
+                                                <a data-toggle="dropdown" class="dropdown-toggle disabled" href="#"><%= link_to "Cart", login_path %></a>
+                                                <ul class="dropdown-menu">
+                                                    <li><%= link_to "Cart", login_path %></li>
+                                                    <li> <%= link_to "History of Cart", login_path, method: :get %></li>
+                                                </ul>
+                                            <% end %>
+
+                                        </li>
+                                        <% if user_signed_in? && current_user.role == 1 %>
```
Comment (vanvtt-0952) [inline app/views/layouts/_header.html.erb:75]: `current_user.role == 1 ` thành `current_user.admin?

**Pull request closed without merging**